tests/storage/hotplug.go

```diff
@@ -963,10 +964,16 @@ var _ = SIGDescribe("Hotplug", func() {
 			vm *v1.VirtualMachine
 		)
 
+		storageClassHostPath := "host-path"
+		immediateBinding := storagev1.VolumeBindingImmediate
+
 		BeforeEach(func() {
+			if tests.Config.ManageStorageClasses {
```

Comment:
I don't think we should care if tests.Config.ManageStorageClasses is set here. We should always create the storage class for this particular test, and always delete it afterwards.